src/settings/secrets/retrievers/environment_variables_retriever.py

```diff
@@ -0,0 +1,10 @@
+import environ
+
+from settings.secrets.retrievers.base_retriever import BaseSecretsRetriever
+
+env = environ.Env()
```

Comment:
I already have one instance. Maybe it's possible to reuse?

Reply:
Decision: this serves a little bit different purpose than the one in `settings.py` file. It's usage here is restricted to this type of retriever and it won't make much sense to add complexity to the creation process. I'm only moving it inside the class.